book.toml

```diff
@@ -11,4 +11,19 @@ edition = "2021"
 
 [output.html]
 no-section-label = true
-git-repository-url = "https://github.com/time-rs/time"
+edit-url-template = "https://github.com/time-rs/book/edit/main/{path}"
+git-repository-url = "https://github.com/time-rs/book/tree/main/"
+
+[output.html.playground]
+editable = true
+runnable = false
+line-numbers = true
+
+[output.html.search]
+boost-hierarchy = 2
+boost-paragraph = 1
+boost-title = 2
+expand = true
+heading-split-level = 2
+limit-results = 20
+use-boolean-and = true
```

Comment:
A quick search shows that a few of these are the default values. Either way, what's the motivation behind changing the defaults?

Reply:
Laziness on my part, repurposing something I drag around. I'll remove the block.